activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -568,7 +568,7 @@ def rename_table(table_name, new_name)
       #   Set to +:cascade+ to drop dependent objects as well.
       #   Defaults to false.
       # [<tt>:if_exists</tt>]
-      #   Set to +true+ to make drop table command fail safe when table does not exists.
+      #   Set to +true+ to make drop table command fail safe when table does not exist.
```

Comment:
Actually can we improve the parity with the `temporary` option below by changing it to `Set to +true+ to only drop the table if it exists.`?